src/course/syllabus/apprenticeship/client-side-app/schedule.md

```diff
@@ -2,52 +2,9 @@
 layout: schedule
 schedule:
   monday:
-    - name: Intro presentation
-      start: 10:10
-      end: 10:30
-      url: https://fac-slides.netlify.app/slides/spa-react/
-    - name: ES Modules intro
-      start: 10:30
-      end: 11:00
-      url: /workshops/es-modules
-      type: workshop
-    - name: DOM rendering
-      start: 11:00
-      end: 13:00
-      url: /workshops/dom-rendering
-      type: workshop
-    - name: React components
-      start: 14:00
-      end: 15:30
-      url: /workshops/react-components
-      type: workshop
-    - name: React state & effects
-      start: 15:30
-      end: 17:45
-      url: /workshops/react-state-effects
-      type: workshop
   tuesday:
-    - name: React forms
-      start: 10:00
-      end: 13:00
-      url: /workshops/react-forms/
-      type: workshop
-    - name: TFB - Discovery
-      start: 14:00
-      end: 15:30
-    - name: React fetching data
-      start: 15:30
-      end: 17:45
-      url: /workshops/react-fetch
-      type: workshop
-  wednesday:
-    - name: React class challenge
-      start: 10:00
-      end: 11:00
-      url: /workshops/react-refactor-classes/
-      type: challenge
-    - name: Project
-      start: 11:00
-      end: 13:00
-      type: project
+    - name: Choose a project idea
+      start: 15:45
+      end: 16:00
+      url: https://github.com/fac25/final-project-ideas/issues
```

Comment:
```suggestion
```

This will need to be updated for every cohort. It may be easier not to link to a repo

Reply:
cool cool